lib/bot.js

```diff
@@ -102,6 +102,11 @@ class Bot {
       this.sendToSlack(author, to, formattedText);
     });
 
+    this.ircClient.on('topic', (channel, topic, nick) => {
+      this.sendToSlack(this.nickname, channel, `*${nick}* changed the topic`);
```

Comment:
Sending the changed topic to Slack as a message is definitely good default behavior though. Maybe the message should include the channel name too? `<username> changed the topic of the IRC channel #chat to "<topic>"`